src/components/icons/copy/index.js

```diff
@@ -1,9 +1,8 @@
 import { h } from "preact";
 
-const Copy = () => (
-  <svg width="15" height="15" fill="none" viewBox="0 0 15 15">
+const Copy = ({active}) => (
+  <svg className={active ? "copy active" : "copy"} width="15" height="15" fill="none" viewBox="0 0 15 15">
```

Comment:
is this the correct way to do this? @reznord

Reply:
Yes, this is how we normally change the colors of the svg.

Reply:
and regarding the JS? and passing in the state?